src/Mvc/test/Mvc.FunctionalTests/Microsoft.AspNetCore.Mvc.FunctionalTests.csproj

```diff
@@ -59,30 +48,4 @@
-      <_PublishFiles Include="$(ArtifactsBinDir)ApplicationModelWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\ApplicationModelWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)BasicWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\BasicWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)ControllersFromServicesWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\ControllersFromServicesWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)CorsWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\CorsWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)ErrorPageMiddlewareWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\ErrorPageMiddlewareWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)FilesWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\FilesWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)FormatterWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\FormatterWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)GenericHostWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\GenericHostWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)HtmlGenerationWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\HtmlGenerationWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)Mvc.RoutingWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\Mvc.RoutingWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)MvcSandbox\$(Configuration)\$(DefaultNetCoreTargetFramework)\MvcSandbox.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)RazorBuildWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\RazorBuildWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)RazorPagesWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\RazorPagesWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)RazorWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\RazorWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)SecurityWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\SecurityWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)SimpleWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\SimpleWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)TagHelpersWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\TagHelpersWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)VersioningWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\VersioningWebSite.deps.json" />
-      <_PublishFiles Include="$(ArtifactsBinDir)XmlFormattersWebSite\$(Configuration)\$(DefaultNetCoreTargetFramework)\XmlFormattersWebSite.deps.json" />
-    </ItemGroup>
-    <Copy SourceFiles="@(_PublishFiles)" DestinationFolder="$(PublishDir)" />
-  </Target>
-
```

Comment:
What was the magic behind removing this, we simply don't need it anymore? (We used to need it for application parts and Razor runtime compilation and we use the API within the WAF for something but can't remember where/why)

Reply:
These weren't here in the MVC Functionals csproj until my last PR, I just added them to copy over the deps file to get them to the publish directory for helix to pickup, that copy logic is now just in the build task, so this is just reverting the hack from the last PR

Reply:
Release/Debug + Environment tag helper

Reply:
Ok I will follow up in a 3rd PR just looking at restoring those 2-3 skipped tests since this one is basically ready to go